cartesi-rollups/challenges/overview.md

```diff
@@ -0,0 +1,84 @@
+
+### Solution strategy
+
+One potential path to achieve transaction scalability is the introduction of an *Aggregator*, often associated with a Sequencer. This component might enhance the overall efficiency and capacity of the rollup solution. Moreover, an exciting area for exploration is deploying Cartesi on top of other Layer-2 solutions like Optimism and Arbitrum, potentially expanding the options for enhanced transaction scalability.
+
+## Disputes
+
+### Challenge
+
+The **1.0** release lacks support for dispute resolution.
+
+### Solution strategy
+
+From its inception, the Cartesi framework has the potential to integrate dispute resolution features. Cartesi's community-driven iterations have leveraged dispute resolution for [Compute](/compute/overview). Prioritizing other features and improvements, like the Microarchitecture enhancement and recent [research on NxN dispute resolution](https://arxiv.org/abs/2212.12439), the community might delve deeper into this area.
+
+## Large data as input
+
+### Challenge
+
+The current Rollups framework does not support very large data inputs (e.g., from IPFS), limiting DApps based on Cartesi Rollups to rely solely on on-chain data.
+
+### Solution strategy
+
+A promising research area is enhancing Cartesi Rollups' data capabilities. Potential advancements in the Ethereum ecosystem, like [EIP-4844](https://www.eip4844.com/) (proto-danksharding), might make on-chain data more affordable. Moreover, layering Cartesi on top of other rollup solutions can offer data advantages.
+
+Another feature for community consideration could be a mechanism assisting front-end clients in dividing large pieces of data into smaller chunks and reassembling them within the back-end. This approach would be similar to the [Logger](/compute/logger_drive/) feature used in Cartesi Compute.
+
+Additionally, potential future exploration might focus on integrating Rollups with Data Availability solutions like zkPorter or using a Data Availability Committee solution for quorum-based validator sets.
+
+## Composability
+
+### Challenge
+
+The current **v1.0** release lacks composability, which is the ability of multiple DApps to directly interact with each other, for example by having one application directly send inputs to another one without going through the base layer. This would enable seamless communication and data exchange between DApps. The most important use case for this feature would be to allow users to easily and cheaply transfer assets from one DApp to another. 
+
+### Solution strategy
+
+Post the **v1.0** mainnet release, a potential area of focus could be research into addressing composability, vital for enhancing DApp functionality. Meanwhile, interim solutions like leveraging liquidity providers for asset transfers might be considered, enhancing the user experience while juggling development priorities.
```

Comment:
Is there any specific area to focus in the research which can be suggested here?

Reply:
I have reworded this sentence in line with the one above